src/main/java/at/hannibal2/skyhanni/features/misc/PingDisplay.kt

```diff
@@ -0,0 +1,138 @@
+package at.hannibal2.skyhanni.features.misc
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.config.commands.CommandCategory
+import at.hannibal2.skyhanni.config.commands.CommandRegistrationEvent
+import at.hannibal2.skyhanni.config.enums.OutsideSbFeature
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.ProfileJoinEvent
+import at.hannibal2.skyhanni.events.minecraft.ClientDisconnectEvent
+import at.hannibal2.skyhanni.events.minecraft.packet.PacketReceivedEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.renderString
+import net.minecraft.client.Minecraft
+import net.minecraft.network.play.client.C16PacketClientStatus
+import net.minecraft.network.play.server.S01PacketJoinGame
+import net.minecraft.network.play.server.S37PacketStatistics
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+import kotlin.math.abs
+import kotlin.math.round
+
+@SkyHanniModule
+object PingDisplay {
+
+    private val config get() = SkyHanniMod.feature.gui
+
+    private val mc: Minecraft = Minecraft.getMinecraft()
```

Comment:
we dont really do this much elsewhere and even if we did, you only use this once in this file so there is no point in it